--- lineage/libcore/NativeCode.mk	2018-03-31 09:23:42.155643585 +0200
+++ lineage_customstuff/libcore/NativeCode.mk	2018-04-01 14:03:35.882571147 +0200
@@ -79,7 +79,7 @@
 core_c_includes := libcore/include $(LOCAL_C_INCLUDES)
 core_shared_libraries := $(LOCAL_SHARED_LIBRARIES)
 core_static_libraries := $(LOCAL_STATIC_LIBRARIES)
-libart_cflags := $(LOCAL_CFLAGS) -Wall -Wextra -Werror
+libart_cflags := $(LOCAL_CFLAGS) -Wall -Wextra
 core_cppflags += -std=gnu++11 -DU_USING_ICU_NAMESPACE=0
 # TODO(narayan): Prune down this list of exclusions once the underlying
 # issues have been fixed. Most of these are small changes except for
